pallets/compliance-manager/src/lib.rs

```diff
@@ -765,6 +875,83 @@ impl<T: Trait> Module<T> {
         }
         Err(Error::<T>::ComplianceRequirementTooComplex.into())
     }
+
+    /// Helper function for the RPC to know the result of the scope claim (i.e investor does posses the valid `InvestorZKProof` claim).
+    fn get_implicit_condition_result(
+        ticker: &Ticker,
+        from_did_opt: Option<IdentityId>,
+        to_did_opt: Option<IdentityId>,
+        primary_issuance_agent: Option<IdentityId>,
+    ) -> Option<ImplicitRequirementResult> {
+        if Self::implicit_requirements_status(ticker) == ImplicitRequirementStatus::Active {
+            let condition_result = |did_opt: Option<IdentityId>| {
+                did_opt.map_or(false, |did| {
```

Comment:
Should it be `false` or `true` for mint/redeem? I think it should be `true`.

Reply:
We are not minting and redeeming with the help of the compliance manager so it doesn't make sense to have `true`

Reply:
There's no other case when `did_opt` can be none. The function should return the correct result, regardless of the context in which it is called. We might start using this in the future for redemptions or some other use case.

Is there an advantage of using `false` instead of `true`? AFAICT, `false` generates incorrect results.